tcms/testplans/static/testplans/js/get.js

```diff
@@ -156,7 +159,7 @@ function attachEvents(testCases, testPlanId, permissions) {
         $('.js-test-case-menu-delete').click(function(ev) {
             const testCaseId = getCaseIdFromEvent(ev);
             jsonRPC('TestPlan.remove_case', [testPlanId, testCaseId], function(res) {
-                $("div").find(`[data-testcase-pk=${testCaseId}]`).remove();
+                deleteClosestTestCaseRow(ev.target);
```

Comment:
For this function is there any benefit of searching the closest element instead of searching by testCaseId ? 

Also a 1 line function for a 1 line call doesn't seem much DRY.

Reply:
$("div") will search all the div elements on the page. As we added the js-testcase-row in previous PR I prefer to use, its is more specific.

I must remove the tc row on remove and when the status is changes. If I dont use this function I will duplicate this  $(ev.target).closest('.js-testcase-row').remove(); which isnt DRY either.

Reply:
instead of `$(ev.target).closest('.js-testcase-row')` 
use ```$(ev.target).closest(`[data-testcase-pk=${testCaseId}]`)```

^^^ or search by .parent()

This minimizes the search path but makes sure to find exactly the row with the ID we are currently trying to modify. This may fail in the future but there's no way it will find and modify an unintended element.

Reply:
>  will duplicate this $(ev.target).closest('.js-testcase-row').remove(); which isnt DRY either.

IMO 1 line duplication is fine. The tools do not detect anything smaller than 3 lines and being able to immediately see what's happening (in this specific case) is far better than the level of abstraction we get with a 1 liner function. 

If this was a language like C that would probably be inlined by the compiler and the respective function completely ignored.